builtins/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "no-global": 2,
+    "query-selector": 2
```

Comment:
Why not add this to `ampproject/amphtml/.eslintrc`? The previous regex based presubmit check was applied globally.

Reply:
Ads mainly, I don't want them to be linted.

Reply:
In that case, you could add `"query-selector": 2` to `ampproject/amphtml/.eslintrc` and disable the rule with `"query-selector": 0` in just the ads folder.